data/meson.build

```diff
@@ -9,18 +9,9 @@ appstream_file = i18n.merge_file(
 appstreamcli = find_program('appstreamcli', required: false)
 if appstreamcli.found()
   test(
-    'Validate appstream file (with appstreamcli)',
+    'Validate appstream file',
     appstreamcli,
-    args: ['validate', '--no-net', appstream_file]
```

Comment:
It seems like my other comments were demotivating. Sorry!

I was shedding off unrelated stuff.

This part of the PR is actually useful.

Thank you for submitting this PR. Would you like to reopen it?